docs/update_and_migration/upgrade_ibexa_dxp_edition.md

```diff
@@ -0,0 +1,118 @@
+---
+description: Upgrade the edition/flavour of your Ibexa DXP installation.
+---
+
+# Upgrading Ibexa DXP Edition
+
+## Recommendations
+
+It is recommenced that 
+- Patch is up-to-date
+- Minor should also be up-to-date
+- One edition/flavor upgrade at a time, move to the closest upper edition:
+  - oss → content
+  - content → experience
+  - experience → commerce
+
+## Get upper edition dependencies
+
+```bash
+departure=content;
+arrival=experience;
+composer show --no-ansi ibexa/$departure | grep versions; # To check the actual version of the current edition
+version=v4.4.2;
+composer require ibexa/$arrival:$version --with-all-dependencies; # There might be errors while running post-update-cmd scripts
+sed -i '/"ibexa\/$departure"/d' composer.json; # Remove from composer.json the previous edition to ease future updates
+```
+
+## Update configurations before imports
+
+Some `post-update-cmd` errors may occur. Their message can be useful to found out missing configurations.
+
+### 4.4.2, from Content to Experience
+
+Add to config/packages/ibexa.yaml, below `framework:`, the following configurations:
+```yaml
+    rate_limiter:
+        corporate_account_application:
+            policy: 'fixed_window'
+            limit: 1
+            interval: '5 minutes'
+            lock_factory: 'lock.corporate_account_application.factory'
+    lock:
+        default: '%env(LOCK_DSN)%'
+        corporate_account_application: '%env(IBEXA_LOCK_DSN)%'
+```
+
+Still in config/packages/ibexa.yaml, add the siteaccess `corporate` to the list and the following `site_group`: `corporate_group: [corporate]`
+
+Re-run `post-update-cmd` script:
+
+```bash
+composer run-script post-update-cmd;
+```
+
+### 4.4.2, from Experience to Commerce
+
+In config/packages/ibexa.yaml:
+- add the following `site_group`: `storefront_group: [site]`
+- under `ibexa.repositories.default.product_catalog`, add
+  ```yaml
+                  regions:
+                    default: ~
+  ```
+
+## Update DB schema
+
+The composer.json of an edition gives the dependency packages to add to the edition immediately below it.
+This package list help to find the schemas to apply to the already existing database.
+
+```bash
+cd vendor/ibexa/doctrine-schema/ && curl -s https://patch-diff.githubusercontent.com/raw/ibexa/doctrine-schema/pull/17.diff | git apply
+
+departure=content;
```

Comment:
It was adding `ibexa:doctrine:schema:dump-sql` from https://github.com/ibexa/doctrine-schema/pull/17 before it was merged. (And it was missing a `cd -` to come back).

```suggestion
departure=content;
```